subseas_flex_fcst/maproom.py

```diff
@@ -360,9 +428,8 @@ def fcst_tiles(tz, tx, ty, proba, variable, percentile, threshold):
         )
     else:
         obs_ppf = threshold
-
     # Forecast CDF
-    fcst_dof = int(fcst_var.attrs["dof"])
+    fcst_dof = int(dofVar)#int(fcst_var.attrs["dof"])
```

Comment:
as above, could be set as int when first set up, and data reading could call it fcst_dof right away in which case the whole lien can be removed.